src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -19,8 +21,23 @@
     private static final String USER = "sqlcmd";
     private static final String PASSWORD = "sqlcmd";
     private static final String TEST_DB_CONNECTION_URL = "jdbc:postgresql://localhost:5432/";
+
+    private static Connection connection;
+
     private DatabaseManager databaseManager;
-    private String dbName;
+    private static String dbName;
```

Comment:
This line should be in the same block with `private static Connection connection;`

Reply:
Ok, i will fix this